src/main/java/org/broadinstitute/hellbender/tools/exome/ACNVModeller.java

```diff
@@ -72,15 +80,15 @@ public ACNVModeller(final SegmentedModel segmentedModel, final String outputPref
      * Segment files (with filenames specified by {@link ACNVModeller#outputPrefix} are output for the initial
      * model fit and the model fits after each merge iteration as a side effect.
      *
-     * @param sigmaThresholdSegmentMean         threshold number of standard deviations for segment-mean similarity
-     * @param sigmaThresholdMinorAlleleFraction threshold number of standard deviations for minor-allele-fraction similarity
+     * @param intervalThresholdSegmentMean         threshold number of credible intervals for segment-mean similarity
```

Comment:
also, note that the command line --help will tell you that these arguments are Doubles and will give the default values.